css/properties/touch-action.json

```diff
@@ -0,0 +1,215 @@
+{
+  "css": {
+    "properties": {
+      "touch-action": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/touch-action",
+          "support": {
+            "webview_android": {
+              "version_added": "36"
+            },
+            "chrome": {
+              "version_added": "36"
+            },
+            "chrome_android": {
+              "version_added": "36"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "52",
+                "notes": "Not applicable to Firefox platforms that support neither pointer nor touch events."
+              },
+              {
+                "version_added": "29",
+                "version_removed": "52",
```

Comment:
The pref has not been removed yet (though on by default), so this line can be dropped.